src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
```

Comment:
It's a good practice to check if the user exists before attempting to update. Consider adding a check to see if a user with the given `id` exists before updating.